munininfluxdb/grafana.py

```diff
@@ -381,7 +380,7 @@ def create_dashboard(self, dashboardJson):
             r.raise_for_status()
 
 
-if __name__ == "__main__":
+if __name__ == "__main__": # pragma: no cover
```

Comment:
In case you have not worked with the coverage module yet: `pragma: no cover` prevents lines from being checked for coverage. There were some cases where this made sense to add (lines which are deliberately left out from unit-testing).